pkg/app/server/grpcapi/api.go

```diff
@@ -144,8 +144,7 @@ func (a *API) AddApplication(ctx context.Context, req *apiservice.AddApplication
 		ProjectId:        key.ProjectId,
 		GitPath:          gitpath,
 		Kind:             req.Kind,
-		CloudProvider:    req.CloudProvider,
```

Comment:
@khanhtc1202 And this.

Reply:
addressed by https://github.com/pipe-cd/pipecd/pull/3821/commits/0ef103520eb13617d13cada263a4184d0818b067 🙏